WorldBank-Transport/DRIVER
# Add Vietnamese translation files

And also fix Portuguese calendar translations

# Testing
- Add `- { id: 'vi', label: 'Vietnamese', rtl: false }` to the `languages` key in `group_vars/all` and reprovision.
- Load the front-end and verify that the interface can be switched to Vietnamese and that the calendar picker works. Verify that the calendar picker is translated, and that it also is now translated into Portuguese.
